docs/reference/high-availability/cluster-design.asciidoc

```diff
@@ -375,11 +375,11 @@ The cluster will be resilient to the loss of any zone as long as:
 - There are at least two zones containing data nodes.
 - Every index that is not a <<searchable-snapshots,searchable snapshot index>>
   has at least one replica of each shard, in addition to the primary.
-- Shard allocation awareness is configured to avoid concentrating all copies of
-  a shard within a single zone.
+- <<shard-allocation-awareness,Shard allocation awareness>> is configured to 
+  avoid concentrating all copies of a shard within a single zone.
 - The cluster has at least three master-eligible nodes. At least two of these 
-  nodes are not voting-only master-eligible nodes, and they are spread evenly
-  across at least three zones.
+  nodes are not <<voting-only-node,dedicated voting-only master-eligible nodes>>, 
```

Comment:
The original text was correct here, the nodes must not be voting-only master-eligible nodes whether dedicated or otherwise.

```suggestion
  nodes are not <<voting-only-node,voting-only master-eligible nodes>>, 
```

Reply:
ah my bad - bad copy paste 😬